docs/src/services/KeyboardProvider.tsx

```diff
@@ -0,0 +1,32 @@
+// @flow
+import * as React from "react";
+import tinykeys from "tinykeys";
+
+export const KeyboardContext = React.createContext<{
+  isSearchOpened: boolean;
+  setSearchOpened: (val: boolean) => void;
+}>({
+  isSearchOpened: false,
+  setSearchOpened: () => {},
+});
+
+export const KeyboardContextProvider = ({ children }): React.ReactNode => {
+  const [isSearchOpened, setSearchOpened] = React.useState(false);
+
+  React.useEffect(() => {
+    const unsubscribe = tinykeys(window, {
+      "$mod+KeyK": () => {
+        setSearchOpened(prev => !prev);
+      },
+    });
+    return () => unsubscribe();
+  }, [setSearchOpened]);
+
+  return (
+    <KeyboardContext.Provider value={{ isSearchOpened, setSearchOpened }}>
+      {children}
+    </KeyboardContext.Provider>
+  );
+};
+
+export const useKeyboard = () => React.useContext(KeyboardContext);
```

Comment:
Considering that logic in here is strictly intended for search, I'd move it to `components/Search/service.tsx` (or something similar), rename `useKeyboard` and `KeyboardContextProvider` to `useSearch` and `SearchProvider` (`SearchProvider` instead of `SearchContextProvider` is more consistent with how we name other providers) and export both `useSearch` and `SearchProvider` from `components/Search/index.tsx` so that you can easily import them elsewhere.

Unrelated, if you want feel free to also rename `hooks/useDevMode.tsx` to `services/dev-mode.tsx` to maintain the naming convention of when we export both the hook and the context provider. Also, use named export for the `useDevMode` hook instead of default export.